app/assets/v2/css/home.css

```diff
@@ -23,6 +130,7 @@
 }
 
 .banner__h2 {
+  font-size: 2.4rem;
```

Comment:
won't the existing font-size classes work ?

Reply:
No, the font-size in the mockup is different than the existing font-size classes